# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrdao/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Taken from Fedora Core to fix compilation with gcc-4.0.

  - Rene Rebe <rene@exactcode.de>

diff -ur cdrdao-1.2.0/trackdb/CueParser.cc cdrdao-1.2.0.new/trackdb/CueParser.cc
--- cdrdao-1.2.0/trackdb/CueParser.cc	2005-05-09 15:21:28.000000000 -0400
+++ cdrdao-1.2.0.new/trackdb/CueParser.cc	2005-07-18 09:38:54.000000000 -0400
@@ -23,6 +23,8 @@
 
 #include "Cue2Toc.h"
 
+class Toc;
+
 extern Toc *parseToc(const char* tocBuffer, const char *filename);
 
 Toc *parseCue(FILE *fp, const char *filename)
Only in cdrdao-1.2.0.new/trackdb: CueParser.o
Only in cdrdao-1.2.0.new/trackdb: .deps
Only in cdrdao-1.2.0.new/trackdb: DLexerBase.o
diff -ur cdrdao-1.2.0/trackdb/FormatConverter.h cdrdao-1.2.0.new/trackdb/FormatConverter.h
--- cdrdao-1.2.0/trackdb/FormatConverter.h	2005-04-21 22:01:46.000000000 -0400
+++ cdrdao-1.2.0.new/trackdb/FormatConverter.h	2005-07-18 09:38:04.000000000 -0400
@@ -24,6 +24,8 @@
 #include <map>
 #include <list>
 
+class Toc;
+
 #include "TrackData.h"
 #include "Toc.h"
 

